examples/client.rs

```diff
@@ -26,10 +26,18 @@ async fn main() {
     loop {
         match client.progress().await.unwrap() {
             ClientFlowEvent::CommandSent {
+                handle: got_handle,
                 tag,
+            } => {
+                println!("command sent: {got_handle:?}, {tag:?}");
+                assert_eq!(handle, got_handle);
+            }
+            ClientFlowEvent::CommandRejected {
                 handle: got_handle,
+                tag,
+                status,
             } => {
-                println!("command sent: {tag:?}, {handle:?}");
+                println!("command aborted: {got_handle:?}, {tag:?}, {status:?}");
```

Comment:
Previously I called the event `ClientFlowEven::CommandAborted` because it makes it clear that `Command` wasn't sent completely. But then I renamed it to `ClientFlowEvent::CommandRejected` because the RFC is using the term "reject". What do you think is the better name?

Reply:
I prefer "reject" because the server rejecting the command is the reason we need to abort sending it. It somehow captures "more" of what happend.